docs/dictionary/command/play.lcdoc

```diff
@@ -27,18 +27,24 @@ play audioClip "Trust No One" looping
 Example:
 play "/usr/local/clips/music.aiff"
 
+Example:
+# to stop the currently playing audio clip
+play stop
+
 Parameters:
+type (enum):
```

Comment:
I'm not sure - it depends on what stuff is broken specifically :-) I think removal of stuff from the docs has to come with corresponding engine changes to remove the syntax, if it is a specific bit of syntax that doesn't work (so that an error is thrown, for example). If it is just that certain formats are not compatible then that would definitely be a docs change.